data/json/vitrification/overmap_vitrified.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "quiet_farm",
+    "locations": [ "forest" ],
```

```suggestion
    "locations": [ "forest" ],
```
Is there any reason you're exclusively spawning it over forest? This will make it considerably rarer that the occurrences imply, if you're set on it I suggest having exceptions for the dirt_road at least

---

I'm trying to make it spawn under close to identical circumstances as a Rural House, in an effort to disguise it as largely unremarkable until it enters the reality bubble. I'd like the result of entering it to start with a very nasty shock, rather than something you can expect to be a really weird time from a dozen tiles away.

Also cuz the surrounding tiles are forest and my brain slides off the documentation to make those tiles change based on neighbors, right now.